_posts/2024-11-24-Bulk.markdown

```diff
@@ -27,3 +27,15 @@ Later in the week I set off on doing some site cleanup (added an icon for the br
 
 -Git gets easier. That sounded funny, but in reality it isn't just git. Everything gets easier with consistency. It doesn't take long to get the hang of these things. A very refreshing prospect.
 
+## End of Week 3 12/6/24
```

Comment:
IMO, you should put these each into their own post.  Becomes more important if/when you tie in things like disqus, social media, etc.  With a new post, you can "announce" it, have a link that contains only the new things, etc etc.